raw/src/types/message.rs

```diff
@@ -30,6 +30,8 @@ pub struct Message {
     pub edit_date: Option<Integer>,
     /// Kind of the message.
     pub kind: MessageKind,
+    /// The unique identifier of a media message group this message belongs to.
+    pub media_group_id: Option<String>,
```

Comment:
I think this field should be a member of `MessageKind::Photo` and `MessageKind::Video`, what do you think?

Reply:
Just commited some changes, what do you think now?